### PR TITLE
Update go-mysql-server dependency

### PR DESCRIPTION
**Pull request by ajnavarro**
Signed-off-by: Antonio Navarro Perez <antnavper@gmail.com>

 
 
 - [x] I updated the documentation explaining the new behavior if any.
 - [x] I updated CHANGELOG.md file adding the new feature or bug fix.
 - [x] I updated go-mysql-server using `make upgrade` command if applicable.
 - [x] I added or updated examples if applicable.
 - [x] I checked that changes on schema are reflected into the documentation, if applicable.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,12 +11,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Only optimize sorted DISTINCT if the first column on the order by is on the distinct schema. ([#976](https://github.com/src-d/gitbase/issues/976))
 - Removed redundant commit information from BLAME results.
 - Don't abort, just warn, on git.Blame errors.
+- sql: Add length to VARCHAR MySQLTypeName string
```
Comment (carlosms) [inline CHANGELOG.md]:
```suggestion
- sql: Add length to VARCHAR and CHAR MySQLTypeName strings in `SHOW CREATE TABLE` output.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,12 +11,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Only optimize sorted DISTINCT if the first column on the order by is on the distinct schema. ([#976](https://github.com/src-d/gitbase/issues/976))
 - Removed redundant commit information from BLAME results.
 - Don't abort, just warn, on git.Blame errors.
```
Comment (carlosms) [inline CHANGELOG.md]: If BLAME is a new function, does it make sense to add these 2 as fixes over the last release?

**Comment by ajnavarro** [comment]
@agarciamontoro drop/create are not supported on gitbase.

**Comment by erizocosmico** [comment]
@ajnavarro can you rebase?

**Comment by ajnavarro** [comment]
@erizocosmico rebased

**Pull request merged**